src/components/app/details/cdDetails/DeploymentDetailSteps.tsx

```diff
@@ -54,19 +58,38 @@ export default function DeploymentDetailSteps({ deploymentStatus, deploymentAppT
         }
     }, [])
 
-    return deploymentStatus === 'Aborted' ? (
+    const redirectToDeploymentStatus = () => {
+        const newUrl = `${URLS.APP}/${appId}/${URLS.APP_DETAILS}/${envId}${URLS.DEPLOYMENT_STATUS}`
+        history.push(newUrl)
+    }
+
+    return deploymentStatus.toUpperCase() === TIMELINE_STATUS.ABORTED || deploymentStatusDetailsBreakdownData.deploymentStatus === DEPLOYMENT_STATUS.SUPERSEDED ? (
```

Comment:
Please check if we need to change second condition to upper case

Reply:
checked